juriscraper/opinions/united_states/state/nytrial.py

```diff
@@ -0,0 +1,223 @@
+        :returns str: formatted url
+        """
+        if not target_date:
+            return self.base_url
+
+        end = f"_{target_date.year}_{target_date.strftime('%B')}.shtml"
+
+        return self.base_url.replace(".shtml", end)
+
+    def is_court_of_interest(self, court: str) -> bool:
+        """'Other Courts' of NY Reporter consists of 10 different
+        family of sources. Each family has an scraper that inherits
+        from this class and defines a `court_regex` to capture those
+        that belong to its family
+
+        For example
+        "Civ Ct City NY, Queens County" and "Civ Ct City NY, NY County"
+        belong to nycivct family
+
+        :param court: court name
+        :return: true if court name matches
+                family of courts of calling scraper
+        """
+        return bool(re.search(self.court_regex, court))
+
+    def _process_html(self) -> None:
+        """Parses a page's HTML into opinion dictionaries
+
+        :return: None
+        """
+        row_xpath = "//table[caption]//tr[position()>1 and td]"
+        for row in self.html.xpath(row_xpath):
+            court = re.sub(
+                r"\s+", " ", row.xpath("td[2]")[0].text_content()
+            ).strip(", ")
+
+            if not self.is_court_of_interest(court):
+                logger.debug("Skipping %s", court)
+                continue
+
+            url = row.xpath("td[1]/a/@href")[0]
+            name = row.xpath("td[1]/a")[0].text_content()
+            opinion_date = row.xpath("td[3]")[0].text_content()
+            slip_cite = row.xpath("td[4]")[0].text_content()
+            status = "Unpublished" if "(U)" in slip_cite else "Published"
+
+            self.cases.append(
+                {
+                    "name": name,
+                    "date": opinion_date,
+                    "status": status,
+                    "url": url,
+                    "citation": slip_cite,
+                    "child_court": court,
+                }
+            )
+
+    def _get_docket_numbers(self) -> List[str]:
+        """Overriding from OpinionSiteLinear, since docket numbers are
+        not in the HTML and they are required
+
+        We will get them on the extract_from_text stage on courtlistener
+        :return: list of empty strings values
+        """
+        return ["" for _ in self.cases]
+
+    def _download_backwards(self, target_date: date) -> None:
+        """Method used by backscraper to download historical records
+
+        :param target_date: an element of self.back_scrape_iterable
+        :return: None
+        """
+        self.url = self.build_url(target_date)
+
+    def _make_html_tree(self, text: str) -> HtmlElement:
+        """Overrides method from AbstractSite
+        Taken from nyappterm_1st that implements this same site
+        but needs different parsing
+
+        :param text: html as text
+        :returns: html object
+        """
+        return get_html5_parsed_text(text)
```

Comment:
I'm confused on this - it doesnt seem to be needed.  And I dont see it implemented in `nyappterm_1st`

Reply:
I think it can be axed.